### PR TITLE
[ADD] web_onchange_action

### PR DESCRIPTION
**Pull request by hbrunn**
Onchange actions
================

This module allows you to return an action (``ir.actions.*``) from an onchange handler.

This way, you can have much more elaborate message boxes or simply show a whole wizard or do whatever else is possible with actions on the client side.

Usage
=====

Depend on this module and in your onchange handler, add a key ``action`` to the ``warning`` dictionary of the result which contains the definition of an action to execute:

.. code:: python

    {
        'value': ....,
        'domain': ....,
        'warning': {
            'title': 'This will be shown if this module is not available',
            'message': 'Fallback message, see above',
            # this module picks up this key and shows it *instead* of the
            # warning above
            'action': {
                'type': 'ir.actions.act_window',
                'res_model': 'my.wizard',
                ...
            },
    }

The module ``web_ir_actions_act_window_message`` can be quite helpful here.

### CODE REVIEW

File: web_onchange_action/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+================
+Onchange actions
+================
+
+This module allows you to return an action (``ir.actions.*``) from an onchange handler.
+
+Whit way, you can have much more elaborate message boxes or simply show a whole wizard or do whatever else is possible with actions on the client side.
```
Comment (LeartS) [inline web_onchange_action/README.rst]: typo: "Whit whay"

**Review by LeartS — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]
As always, cool and elegant.

**Comment by StefanRijnhart** [comment]
@hbrunn Don't forget the typo that @LeartS noticed.

**Comment by StefanRijnhart** [comment]
OMG used inline Github editing for the first time to correct the typo.

**Comment by hbrunn** [comment]
can we merge?

**Comment by StefanRijnhart** [comment]
Merge slightly overdue!

**Comment by paulius-sladkevicius** [comment]
Hi @hbrunn, @StefanRijnhart do you know if something similar exists on the v13/14 or best to try to port this one?

**Comment by StefanRijnhart** [comment]
Nope, I think Odoo missed this one when it looked for good OCA modules to reimplement in the core.

**Comment by paulius-sladkevicius** [comment]
> Nope, I think Odoo missed this one when it looked for good OCA modules to reimplement in the core.

@StefanRijnhart thanks, in this case we'll try to port it.

**Pull request merged**